### PR TITLE
Feature/dx 443 completeurl update description

### PR DESCRIPTION
**Pull request by NullableInt**
Fixes complete url description and redirect page tables for mobile pay

### CODE REVIEW

File: _includes/purchase.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Content-Type: application/json
...
+|  ✔︎︎︎︎︎  | └─➔&nbsp;`completeUrl`                | `string`      | The URL that Swedbank Pay will redirect back to when the payment page is completed. This does not indicate a successful payment, only that it has reached a completion state. A `GET` request needs to be performed on the payment to inspect it further.                                                                                                                                                                                                                                                                                                                 |
```
Comment (asbjornu) [inline _includes/purchase.md]:
```suggestion
|  ✔︎︎︎︎︎  | └─➔&nbsp;`completeUrl`                | `string`      | The URL that Swedbank Pay will redirect back to when the payer has completed his or her interactions with the payment. This does not indicate a successful payment, only that it has reached a final (complete) state. A `GET` request needs to be performed on the payment to inspect it further.                                                                                                                                                                                                                                                                                                                 |
```

File: checkout/other-features.md [hunk 2]
```diff
@@ -179,15 +179,15 @@ that is used to inform the payee (merchant) of changes or updates made to
...
+| :------: | :------------------ | :------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ✔ ︎︎︎︎︎  | `hostUrls`          | `array`  | The array of URIs valid for embedding of Swedbank Pay Hosted Views.                                                                                                                                                                                             |
+|  ✔︎︎︎︎︎  | `completeUrl`       | `string` | The URL that Swedbank Pay will redirect back to when the payment page is completed. This does not indicate a successful payment, only that it has reached a completion state. A `GET` request needs to be performed on the payment order to inspect it further. |
```
Comment (asbjornu) [inline checkout/other-features.md]:
```suggestion
|  ✔︎︎︎︎︎  | `completeUrl`       | `string` | The URL that Swedbank Pay will redirect back to when the payer has completed his or her interactions with the payment. This does not indicate a successful payment, only that it has reached a final (complete) state. A `GET` request needs to be performed on the payment order to inspect it further. |
```

File: checkout/other-features.md [hunk 3]
```diff
@@ -233,18 +233,18 @@ Content-Type: application/json
...
+| └➔&nbsp;`id`                | `string` | The relative URI to the `urls` resource.                                                                                                                                                                                                                        |
+| └➔&nbsp;`hostsUrl`          | `string` | An array of the whitelisted URIs that are allowed as parents to a Hosted View, typically the URI of the web shop or similar that will embed a Hosted View within it.                                                                                            |
+| └➔&nbsp;`completeUrl`       | `string` | The URL that Swedbank Pay will redirect back to when the payment page is completed. This does not indicate a successful payment, only that it has reached a completion state. A `GET` request needs to be performed on the payment order to inspect it further. |
```
Comment (asbjornu) [inline checkout/other-features.md]:
```suggestion
| └➔&nbsp;`completeUrl`       | `string` | The URL that Swedbank Pay will redirect back to when the payer has completed his or her interactions with the payment. This does not indicate a successful payment, only that it has reached a final (complete) state. A `GET` request needs to be performed on the payment order to inspect it further. |
```

**Comment by NullableInt** [comment]
I did a search-replace. Is it good?

**Pull request merged**